contracts/scripts/deploy.sh

```diff
@@ -248,15 +248,15 @@ cargo contract build --release
 cd "$CONTRACTS_PATH"/wrapped_azero
 cargo contract build --release
 
-# --- DEPLOY ACCESS CONTROL CONTRACT
+# # --- DEPLOY ACCESS CONTROL CONTRACT
 
 cd "$CONTRACTS_PATH"/access_control
 
 ACCESS_CONTROL_CODE_HASH=$(cargo contract upload --url "$NODE" --suri "$AUTHORITY_SEED")
 ACCESS_CONTROL_CODE_HASH=$(echo "$ACCESS_CONTROL_CODE_HASH" | grep hash | tail -1 | cut -c 14-)
-ACCESS_CONTROL=$(cargo contract instantiate --url "$NODE" --constructor new --suri "$AUTHORITY_SEED" --skip-confirm)
-ACCESS_CONTROL=$(echo "$ACCESS_CONTROL" | grep Contract | tail -1 | cut -c 14-)
-ACCESS_CONTROL_PUBKEY=$(docker run --rm --entrypoint "/bin/sh" "${NODE_IMAGE}" -c "aleph-node key inspect $ACCESS_CONTROL" | grep hex | cut -c 23- | cut -c 3-)
+ACCESS_CONTROL=$(cargo contract instantiate --url "$NODE" --constructor new --suri "$AUTHORITY_SEED" --skip-confirm --output-json)
+ACCESS_CONTROL=$(echo "$ACCESS_CONTROL" | jq -r '.contract')
+ACCESS_CONTROL_PUBKEY=$("$ROOT"/target/release/aleph-node key inspect $ACCESS_CONTROL | grep hex | cut -c 23- | cut -c 3-)
```

Comment:
Just FYI @fbielejec , I dropped the usage of docker here and use node binary directly.

Reply:
but why :fearful:

Reply:
No dependency on the docker here anymore and you have to compile `aleph-node` anyway to run these tests and the docker image wasn't available for me locally anyway. 

Although, now that I think of it, you're calling `deploy.sh` in contexts where there might be no `aleph-node` binary available 🤔 I'll be fine with reverting this change and having local version of it for myself.